.ci_support/linux_64_python3.12.____cpython.yaml

```diff
@@ -1,27 +0,0 @@
-c_compiler:
-- gcc
-c_compiler_version:
-- '13'
-c_stdlib:
-- sysroot
-c_stdlib_version:
-- '2.17'
-cdt_name:
-- cos7
-channel_sources:
-- conda-forge
-channel_targets:
-- conda-forge main
-docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
-pin_run_as_build:
-  python:
-    min_pin: x.x
-    max_pin: x.x
-python:
-- 3.12.* *_cpython
-target_platform:
-- linux-64
```

Comment:
It looks like several of the `linux_64_*` variants got deleted (like this one). Is this expected?

Reply:
Yeah, they got merged into one `use_noarch: false` variant that produces a conda package for each variant. Not exactly sure why.

Reply:
Thanks Isuru! 🙏

Ah ok. Can see that here

https://github.com/conda-forge/coverage-feedstock/blob/8d9a1e303b4dae56ffa84a97516b812b1a98893f/.ci_support/linux_64_use_noarchfalse.yaml#L21-L30

It's interesting the `noarch` case has the same behavior. Though admittedly conda-build may not go through enough rendering phases to know this can be dropped

https://github.com/conda-forge/coverage-feedstock/blob/8d9a1e303b4dae56ffa84a97516b812b1a98893f/.ci_support/linux_64_use_noarchtrue.yaml#L21-L30